package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-tree-shaking": "^1.7.3",
     "gl": "^4.2.2",
     "mkdirp": "^0.5.1",
+    "mjolnir.js": "^2.1.2",
```

Comment:
Why would we have mjolnir.js in the main package.json? Shouldn't this be an optional dependency for examples.

Reply:
We load the examples in the browser render tests. They would crash if mjolnir is not found from the test directory.